akka-stream/src/main/scala/akka/stream/impl/FlexiMergeImpl.scala

```diff
@@ -127,30 +117,30 @@ private[akka] class FlexiMergeImpl(_settings: MaterializerSettings,
 
   nextPhase(TransferPhase(precondition) { () ⇒
     behavior.condition match {
-      case read: ReadAny ⇒
+      case read: ReadAny[t] ⇒
```

Comment:
I avoid existential types wherever possible, in particular because it will be “erased” to `Any` in such a match. Putting in a lowercase name binds the (unknown) type to a name and thus enables normal type checking.